### PR TITLE
Adding group relationship with team

### PR DESCRIPTION
**Pull request by psaffaie**
adding documentation for what the relationship between a group and a team are

### CODE REVIEW

**Review by FaithOmbongi — APPROVED** [review]
Thank you @psaffaie for closing this doc gap.

File: api-reference/v1.0/resources/group.md [hunk 1]
```diff
@@ -194,6 +194,8 @@ This resource supports:
...
 | sites                    | [site](site.md) collection                                            | The list of SharePoint sites in this group. Access the default site with /sites/root.                                                                                                                                                                                                                                                                                                                                                                                        |
+| team                    | [channel](channel.md) collection                                            | Every team is associated with a [Microsoft 365 group](../resources/group.md). The group has the same ID as the team - for example, `/groups/{id}/team` is the same as `/teams/{id}`. For more information about working with groups and members in teams, see [Use the Microsoft Graph REST API to work with Microsoft Teams](teams-api-overview.md). |
```
Comment (Lauragra) [inline api-reference/v1.0/resources/group.md:197]: Is this the description that we want, or should the description be similar to that of the other relationships - i.e. "The team associated with this group."

Comment (Lauragra) [inline api-reference/v1.0/resources/group.md:197]: I noticed that this relationship is also on the resource in beta but is not documented in the beta topic. Please update that topic as well. Also, adding a new relationship merits an entry in the changelog; please add that to this PR.

Comment (FaithOmbongi) [reply api-reference/v1.0/resources/group.md:197]: Hi @Lauragra, this is a change to resolve the out-of-sync schema/docs issues. We're avoiding new changelogs for these properties and relationships that may have existed in the schema for a long time but lacked related docs.

**Comment by psaffaie** [comment]
@FaithOmbongi In this PR I updated both the v1 and beta docs! Let me know if things look good on your end :)

**Comment by FaithOmbongi** [comment]
Hi @psaffaie, I pushed a commit to remove an extra line in the table and looks good. Added to the merge queue.

**Pull request merged**